pkgs/os-specific/linux/dpdk-kmods/default.nix

```diff
@@ -2,20 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "dpdk-kmods";
-  version = "2022-08-29";
+  version = "2023-02-05";
```

Comment:
Also it's not this issue is not introduced by this PR.
Could you add `broken = kernel.isHardened` to meta? Than we no longer need to test the build for this kernel. Thanks!

Reply:
Done, though this is what happens on master:

```
$ nix-build -A linuxKernel.packages.linux_lqx.dpdk-kmods
[...]
error: builder for '/nix/store/5s5v974a5m9vh2yf4611ls1ha7ama76w-linux-config-6.7.9.drv' failed with exit code 255;
       last 25 log lines:
       > GOT: * Runtime Testing
       > GOT: *
       > QUESTION: Runtime Testing, NAME: RUNTIME_TESTING_MENU, ALTS: N/y/?, ANSWER: n
       > GOT: n
       > QUESTION: Memtest, NAME: MEMTEST, ALTS: Y/n/?, ANSWER: y
       > GOT: y
       > QUESTION: Microsoft Hyper-V driver testing, NAME: HYPERV_TESTING, ALTS: N/y/?, ANSWER:
       > GOT:
       > GOT: *
       > GOT: * Rust hacking
       > GOT: *
       > GOT: #
       > GOT: # configuration written to .config
       > GOT: #
       > GOT: make[1]: Leaving directory '/build/source/build'
       > GOT: make: Leaving directory '/build/source'
       > warning: unused option: CFS_BANDWIDTH
       > warning: unused option: MLX5_CORE_EN
       > warning: unused option: NUMA_BALANCING
       > warning: unused option: PSI
       > warning: unused option: RT_GROUP_SCHED
       > warning: unused option: SCHED_AUTOGROUP
       > warning: unused option: SCHED_CORE
       > error: unused option: UCLAMP_TASK
       > error: unused option: UCLAMP_TASK_GROUP
       For full logs, run 'nix log /nix/store/5s5v974a5m9vh2yf4611ls1ha7ama76w-linux-config-6.7.9.drv'.
error: 1 dependencies of derivation '/nix/store/mrkklqr7jaddazda0wl1gcy8i35hwqyi-linux-6.7.9.drv' failed to build
error: 1 dependencies of derivation '/nix/store/h1rdrcmr2ysjacf3nd2cvcxh9l388r92-dpdk-kmods-2022-08-29.drv' failed to build
```

which may be related to #296867, but I haven't investigated further.

Reply:
Also note that it does build for some other hardened kernels